.travis.yml

```diff
@@ -1,6 +1,3 @@
-# Use older version of Trusty VM due to issues in the updated one
```

Comment:
this was a workaround before when we were having larger logs that was causing issues in newer travis VM versions, but now we've removed the verbose output I believe it is no longer needed